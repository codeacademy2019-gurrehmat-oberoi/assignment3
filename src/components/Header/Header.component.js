import React from 'react';
import './Header.css';

const Header = () => (
  <header>
    <div className="hr" />
    <div className="headerContent">
      <div className="logo">BS</div>
      <div className="headerText">The Book Shelf</div>
    </div>
    <div className="hr" />
  </header>
);

export default Header;
