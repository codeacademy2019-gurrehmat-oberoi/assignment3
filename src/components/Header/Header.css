header {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-content: center;
}

header .hr {
  width: 100%;
  height: 5px;
  background-color: black;
  align-self: center;
}

header .headerContent {
  display: flex;
  flex-direction: column;
  align-items: center;
  flex-grow:1;
  width:30%;
  min-width: fit-content;
  margin: 10px;
}

header .headerContent .logo {
  padding: 5px;
  border: 2px solid;
  border-radius: 50%;
  font-size: 30px;
}

header .headerContent .headerText {
  font-size: 30px;

}