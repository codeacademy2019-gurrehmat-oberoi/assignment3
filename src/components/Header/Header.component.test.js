import renderer from 'react-test-renderer';
import React from 'react';
import Header from './Header.component';

describe('Header', () => {
  it('should render a header', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
