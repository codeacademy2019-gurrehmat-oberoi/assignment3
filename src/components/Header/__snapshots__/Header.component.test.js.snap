// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render a header 1`] = `
<header>
  <div
    className="hr"
  />
  <div
    className="headerContent"
  >
    <div
      className="logo"
    >
      BS
    </div>
    <div
      className="headerText"
    >
      The Book Shelf
    </div>
  </div>
</header>
`;
